README.md

```diff
@@ -9,7 +9,15 @@
 Currently there are three tools in the katydid toolkit:
 
   * Relapse: a regular expression type language for trees that matches up to 1000000s of records per second, 
-  * A collection of parsers (protobuf, json, xml, reflected go structures) which are easily extendable, and
+  * A collection of parsers (protobuf, json, xml, reflected go structures, yaml*) which are easily extendable, and
   * A collection of encoders (protobuf, json, xml, reflected go structures) which are useful for dynamic transcoding.
 
 [more...](http://katydid.github.io)
+
+*The YAML parser implementation is in progress and has the following knwown limitations
```

Comment:
Maybe let's create a .md file in the yaml folder and add this there or even as a comment in the godoc of the yaml.go file, what do you think?

Reply:
I agree, this leaves the readme clean. I'll go with the .md file in the yaml folder.